dml/import_schools.sql

```diff
@@ -40,7 +40,7 @@ insert into schools (
 
 select
 	sr."URN"::integer,
-	sr."UKPRN"::integer,
+	CASE WHEN sr."UKPRN" != '' THEN sr."UKPRN"::integer ELSE NULL END as UKPRN,
```

Comment:
Looks good - perhaps easier to read when inverted?

```suggestion
   case -- ukprn
   when (sr."UKPRN" is null or sr."UKPRN" = '')
           then null
   else
           sr."UKPRN"::integer
   end,
```

Also we don't need the `as` here as we're in an `insert`.

Reply:
Not sure if the GitHub web editor respects tabs 🤔